### PR TITLE
Add SXG testing instructions

### PR DESCRIPTION
**Pull request by twifkak**
Adds testing instructions to the "How to set up signed exchanges" document, so publishers can verify the setup is correct.

### CODE REVIEW

**Comment by twifkak** [comment]
cc @kumagi

**Comment by kaycebasques** [comment]
@kumagi please leave an explicit LGTM when you're done with your review and then I'll take over from there

**Comment by kumagi** [comment]
LGTM. Great.

File: src/site/content/en/blog/how-to-set-up-signed-http-exchanges/index.md [hunk 1]
```diff
@@ -259,6 +259,16 @@ server {
 }
 ```
 
+## Step 6: Test
+
+Use the [dump-signedexchange tool](https://github.com/WICG/webpackage/tree/master/go/signedexchange) to test that the SXGs being served are correct. Ensure that no errors are reported, and verify that the headers and body are as expected.
+
```
Comment (kaycebasques) [inline src/site/content/en/blog/how-to-set-up-signed-http-exchanges/index.md:265]:
```suggestion
## Step 6: Test

Use the [dump-signedexchange tool](https://github.com/WICG/webpackage/tree/master/go/signedexchange)
to test that the SXGs being served are correct, ensure that no errors are reported, and verify that the headers
and body are as expected.

```

**Comment by twifkak** [comment]
Thanks!

**Pull request merged**